.changeset/thick-dots-sit.md

```diff
@@ -1,5 +1,5 @@
 ---
-"wrangler": patch
+"wrangler": minor
```

Comment:
as discussed with @CarmenPopoviciu maybe this should be considered a minor as it adds support for assets?